src/signals/incident-management/containers/IncidentDetail/components/MetaList/components/ChangeValue/index.js

```diff
@@ -52,18 +53,17 @@ const ChangeValue = ({
   const form = useMemo(
     () =>
       FormBuilder.group({
-        input: [get(incident, valuePath), Validators.required],
+        input: [getValue(get(incident, valuePath)), Validators.required],
```

Comment:
I'd say make `get` the default, for which a specific implementation can be provided through the props. Otherwise it gets a bit to layered and complicated, and you're still missing out on flexibility.

```JavaScript
const ChangeValue = ({
  ...,
  get = lodashGet,
  set = lodashSet,
  ...,
}) => {
  ...
  input: [get(incident, valuePath), Validators.required],
  ...
};
```

Reply:
Good suggestion, I have replaced only the get function though. Replacing of the set function was not needed.